profileTemplate

```diff
@@ -29,25 +29,24 @@
 =cut
 
 
-$data_dir       = '/PATH/TO/DATA/location';     # where the data goes
-$bin_dir        = '/PATH/TO/BIN/location';      # where the imaging scripts go
-$prefix         = "project";                    # study prefix or study name
-$mail_user      = "yourname\@example.com";      # who will be notified
-$get_dicom_info = "/PATH/TO/get_dicom_info.pl"; # path to get_dicom_info.pl
-$horizontalPics = 1;
-$no_nii         = 0;                            # 0=create nifti files, 1=don't create nifti files
+$data_dir        = '/PATH/TO/DATA/location';     # where the data goes
+$bin_dir         = '/PATH/TO/BIN/location';      # where the imaging scripts go
+$prefix          = "project";                    # study prefix or study name
+$mail_user       = "yourname\@example.com";      # who will be notified
+$get_dicom_info  = "/PATH/TO/get_dicom_info.pl"; # path to get_dicom_info.pl
+$horizontalPics  = 1;
+$no_nii          = 0;                            # 0=create nifti files, 1=don't create nifti files
+$PatientID_regex = '([^_]+)_(\d+)_([^_]+)'       # regular expression to parse PatientName or PatientID
```

Comment:
@spaiva 
For the regex, I had to modify it to: ([^_]+)_(\d+)_([^_]+)|(lego_phantom)|(living_phantom) to allow insertion of phantom scans as well. Several projects use them and it is how they are labelled.

Other than that, all good! :) Let me know when you make the change and I'll check the PassedCodeReview.